### PR TITLE
Fix map component overflow on smaller screens

### LINKED ISSUE #95: Map component on public event page overflows
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**

image

image

The Map overflows and adress disappears with the `address` heading coming onto the map itself.

**Expected behavior:**

It should be a stackable grid so that the address does not flow in to the map component on smaller screens
image

**Steps to reproduce:**

View it on mobile(both landscape and portrait)

I will work on this.

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

image
Over flow of map component

#### Changes proposed in this pull request:

image

image

Fixes #95

### CODE REVIEW

File: tests/integration/components/public/event-map-test.js [hunk 1]
```diff
@@ -9,5 +9,5 @@ let event = Object.create({ latitude: 37.7833, longitude: -122.4167, locationNam
 test('it renders', function(assert) {
   this.set('event', event);
   this.render(hbs `{{public/event-map event=event}}`);
-  assert.equal(this.$('.address p').text(), 'Sample event location address');
+  assert.ok(this.$().html().trim().includes('Sample event location address'));
```
Comment (CosmicCoder96) [inline tests/integration/components/public/event-map-test.js]: For uniformity with other component tests.

**Comment by CosmicCoder96** [comment]
@niranjan94 Please review.

File: tests/integration/components/public/event-map-test.js (hunk 1 above)
Comment (niranjan94) [inline tests/integration/components/public/event-map-test.js]: No @CosmicCoder96 . Ideally this is how we want all tests to be. We want to actually check if the data is correctly displayed. Please revert this back to how it was.

**Comment by CosmicCoder96** [comment]
@niranjan94 Will keep that in mind, have made the changes please review.

**Pull request merged**